### PR TITLE
Update tradfri.markdown

### PR DESCRIPTION
**Pull request by Azimoth**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/tradfri.markdown [hunk 1]
```diff
@@ -48,3 +48,6 @@ Do not use the `api_key` variable. The key is only needed once at initial setup.
 Please make sure you have `autoconf` installed (`apt-get install autoconf`) if you want to use this component. Also, installing some dependencies might take considerable time (>1h) on slow devices. You might have to use `sudo` when installing `autoconf`.
 
 </p>
+
+## Troubleshooting
```
Comment (sjabby) [inline source/_components/tradfri.markdown]: Please change this into a linkable title:
```
## {% linkable_title Troubleshooting %}
```

Comment (DubhAd) [reply source/_components/tradfri.markdown]: See [the standards guide](https://developers.home-assistant.io/docs/en/documentation_standards.html) for details on linkable titles, etc

**Review by sjabby — APPROVED** [review]

**Review by DubhAd — APPROVED** [review]
👍

**Pull request merged**